package.json

```diff
@@ -23,6 +23,7 @@
   "homepage": "https://github.com/digitalbazaar/bedrock-redis",
   "dependencies": {
     "backo": "^1.1.0",
+    "promise-redis": "0.0.5",
     "redis": "^2.8.0"
```

Comment:
looks like we no longer need this dep?

Reply:
Still need the redis dep. promise-redis just depends on "*" so want to restrict it to what we support.

Reply:
@davidlehn is correct -- and really `promise-redis` should be using peerDeps for that, but whatever.

Reply:
I think it would be a good idea to put that information in the changelog.